guides/doc-Monitoring_Guide/master.adoc

```diff
@@ -0,0 +1,25 @@
+= Monitoring Red Hat Satellite
+:imagesdir: topics/images
+:monitoring-red-hat-satellite:
+:numbered:
+:context: monitoring-guide
+
+include::common/attributes.adoc[]
+include::topics/con_satellite_metrics_overview.adoc[leveloffset=+1]
+include::topics/con_performance-co-pilot.adoc[leveloffset=+1]
```

Comment:
please remove this file - total garbage that adds nothing

Reply:
Only this component/file? It's introduction to PCP followed by introduction to PMDA.